### PR TITLE
PLAT-433: parallel call check semaphore

### PR DESCRIPTION
**Pull request by ivanshibitov**
Add test which checks that we run parallelly no more than semaphore permits

### CODE REVIEW

File: ledger-core/virtual/integration/semaphore_test.go [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+	server.SetMaxParallelism(semaphoreParallelism)
+
+	var (
+		class            = testwallet.GetClass()
+		interferenceFlag = contract.CallIntolerable
+		stateFlag        = contract.CallDirty
+		numParallelExecs = int64(0)
+	)
+
+	syncChan := make(chan bool)
```
Comment (ruz) [inline ledger-core/virtual/integration/semaphore_test.go:67]: use sync point like other tests do

Comment (ruz) [reply ledger-core/virtual/integration/semaphore_test.go:67]: I understand that you can not use sync point

File: ledger-core/virtual/integration/semaphore_test.go [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+	"github.com/insolar/assured-ledger/ledger-core/runner/execution"
+	"github.com/insolar/assured-ledger/ledger-core/runner/requestresult"
+	"github.com/insolar/assured-ledger/ledger-core/testutils/runner/logicless"
+	"github.com/insolar/assured-ledger/ledger-core/virtual/integration/utils"
+	"github.com/insolar/assured-ledger/ledger-core/virtual/testutils"
+)
+
+func TestVirtual_SemaphoreLimitNotExceeded(t *testing.T) {
+	t.Log("C5137")
+	t.Skip("https://insolar.atlassian.net/browse/PLAT-432")
```
Comment (ruz) [inline ledger-core/virtual/integration/semaphore_test.go:29]: this test shouldn't depend on PLAT-432

File: ledger-core/virtual/integration/semaphore_test.go [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+				CallSiteMethod:      objects[i].String(),
+				CallOutgoing:        server.BuildRandomOutgoingWithPulse(),
+			}
+			server.SendPayload(ctx, &pl)
+		}
+	}
+
+	// Wait after every execution to increase probability that multiple executions are in queue
+	for i := 0; i < numObject; i++ {
+		select {
```
Comment (bigbes) [inline ledger-core/virtual/integration/semaphore_test.go:111]: can you replace it with `testutils.WaitSignalsTimed`?

Comment (bigbes) [reply ledger-core/virtual/integration/semaphore_test.go:111]: misread that part

Comment (ivanshibitov) [reply ledger-core/virtual/integration/semaphore_test.go:111]: testutils.WaitSignalsTimed waits reading, I wait for opportunity to write

**Pull request merged**